spec/models/course/material/folder_ability_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+require 'cancan/matchers'
```

Comment:
I didn't see this, sorry. This should be a support library, or include this inside rails_helper.